app/containers/RouteReviewPage/index.js

```diff
@@ -35,15 +44,53 @@ export class RouteReviewPage extends React.PureComponent { // eslint-disable-lin
             <div className="col-md-8 px-0">
               <div className="container">
                 <Header />
-                <Breadcrumbs breadcrumbData={[{ link: 'google.com', text: 'Denali National Park' }]} />
+
+                {/*Write report*/}
+                <PageSection title="Write a Trip Report">
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                {/*Location section, need to downsize & add new container w/ locations*/}
+                <PageSection title="Location">
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                {/*Rating */}
+                <PageSection title="Rating">
+                  <Stars className="d-inline-block pl-2" value={0.0} editable={true} /> 
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                {/*Report */}
+                <PageSection title="Report">
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                {/*Beta*/}
+                <PageSection title="Beta">
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                {/*Add Photo/Video */}
+                <PageSection title="Add Photo/Video">
+                  <BorderBottomDiv className="pb-2">  
+                  </BorderBottomDiv>
+                </PageSection>
+
+                <BlueBox  />
```

Comment:
I think we should not make the submit button a separate component. It will only be used on this page I think, so there's no advantage in sharing in between other components.

Go ahead and inline the button here, no need for that BlueBox component